WordPress/src/main/java/org/wordpress/android/util/config/DashboardPersonalizationFeatureConfig.kt

```diff
@@ -0,0 +1,20 @@
+package org.wordpress.android.util.config
+
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.annotation.Feature
+import javax.inject.Inject
+
+@Feature(DashboardPersonalizationFeatureConfig.DASHBOARD_PERSONALIZATION_REMOTE_FIELD, true)
```

Comment:
Hey @zwarm 
I think this value that is passed on Annotation should be `false` instead of `true`